base/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM golang:1.12.9-stretch AS builder
+
+MAINTAINER free5GC <support@free5gc.org>
+
+ENV GO111MODULE=off
+
+RUN apt-get update
+RUN apt-get -y install gcc cmake autoconf libtool pkg-config libmnl-dev libyaml-dev
+RUN apt-get -y install netcat tcpdump iproute2 netbase
+RUN apt-get clean
+
+# Get Free5GC
+RUN cd $GOPATH/src \
+    && git clone https://github.com/free5gc/free5gc.git \
+    && cd free5gc \
+    && git submodule update --init --jobs `nproc` \
+    && ./install_env.sh -j `nproc`
+
+# Build NF (AMF, AUSF, N3IWF, NRF, NSSF, PCF, SMF, UDM, UDR)
+RUN cd $GOPATH/src/free5gc/src \
+    && for d in * ; do if [ -f "$d/$d.go" ] ; then go build -o ../bin/"$d" -x "$d/$d.go" ; fi ; done ;
+
+# Build UPF
+#RUN go get -u -v "github.com/sirupsen/logrus"
+RUN cd $GOPATH/src/free5gc/src/upf \
+    && mkdir -p build \
+    && cd build \
+    && cmake .. \
+    && make -j `nproc`
+
+# Build WebUI
+RUN cd $GOPATH/src/free5gc/webconsole \
+    && go build -o webui -x server.go
+
+# Alpine is used for debug purpose. You can use scratch for a smaller footprint.
+FROM alpine
+
+WORKDIR /free5gc
+RUN mkdir -p config/ support/TLS/
+
+# Copy executables
+COPY --from=builder /go/src/free5gc/bin/* ./
+COPY --from=builder /go/src/free5gc/src/upf/build/bin/* ./
+COPY --from=builder /go/src/free5gc/webconsole/webui ./
```

Comment:
If we want to run webui, we should also copy free5gc/webconsole/public folder for webui's frontend.

Reply:
Thanks. I updated it but I still get 404. I'll keep debugging.

Reply:
I think the folder name outside webui binary must be `webconsole`

Reply:
This is because webui will find public folder inside webconsole

https://github.com/free5gc/webconsole/blob/0f89755bd497a2dd2c6ae813c4e6f8411db1df65/backend/webui_service/middleware.go#L11

Reply:
Perfect :100: Now it works like a charm :+1: